 yanshee:
  # Publish all joint states -----------------------------------
  joint_state_controller:
    type: joint_state_controller/JointStateController
    publish_rate: 100

  head_position_controller:
    type: position_controllers/JointPositionController
    joint: head_joint

  left_arm1_position_controller:
    type: position_controllers/JointPositionController
    joint: left_arm1_joint
  left_arm2_position_controller:
    type: position_controllers/JointPositionController
    joint: left_arm2_joint
  left_arm3_position_controller:
    type: position_controllers/JointPositionController
    joint: left_arm3_joint

  right_arm1_position_controller:
    type: position_controllers/JointPositionController
    joint: right_arm1_joint
  right_arm2_position_controller:
    type: position_controllers/JointPositionController
    joint: right_arm2_joint
  right_arm3_position_controller:
    type: position_controllers/JointPositionController
    joint: right_arm3_joint

  #left_leg1_position_controller:
    #type: position_controllers/JointPositionController
    #joint: left_leg1_joint
  #left_leg2_position_controller:
    #type: position_controllers/JointPositionController
    #joint: left_leg2_joint
  #left_leg3_position_controller:
    #type: position_controllers/JointPositionController
    #joint: left_leg3_joint
  #left_leg4_position_controller:
    #type: position_controllers/JointPositionController
    #joint: left_leg5_joint
  #left_leg5_position_controller:
    #type: position_controllers/JointPositionController
    #joint: left_leg5_joint

  #right_leg1_position_controller:
    #type: position_controllers/JointPositionController
    #joint: right_leg1_joint
  #right_leg2_position_controller:
    #type: position_controllers/JointPositionController
    #joint: right_leg2_joint
  #right_leg3_position_controller:
    #type: position_controllers/JointPositionController
    #joint: right_leg3_joint
  #right_leg4_position_controller:
    #type: position_controllers/JointPositionController
    #joint: right_leg5_joint
  #right_leg5_position_controller:
    #type: position_controllers/JointPositionController
    #joint: right_leg5_joint
